resources/translations/fi.edn

```diff
@@ -124,7 +124,7 @@
                           :returned "Palautettu muokattavaksi"
                           :submitted "Haettu"
                           :unknown "Tuntematon"}
-                 :intro [:div.intro [:p "Täyttämällä ja lähettämällä tämän hakemuksen, kaikki hakijat voivat saada oikeuden haettuihin resursseihin. Kaikkien on sen lisäksi hyväksyttävä kaikki käyttöehdot viimeistään hakemuksen käsittelyn jälkeen saadakseen käyttöoikeudet."]]
+                 :intro [:div.intro [:p "Täyttämällä ja lähettämällä tämän hakemuksen, kaikki hakijat voivat saada oikeuden haettuihin resursseihin. Kaikkien hakijoiden on sen lisäksi hyväksyttävä kaikki käyttöehdot viimeistään hakemuksen käsittelyn jälkeen saadakseen käyttöoikeudet."]]
```

Comment:
The comma in the first sentence is actually recommended against, see [here](http://www.kielitoimistonohjepankki.fi/ohje/95).

A proposal: "Kaikkien hakijoiden on sen lisäksi hyväksyttävä [...]" -> "Kunkin hakijan on lisäksi hyväksyttävä [...]" (for brevity and reduced tautophony)

Reply:
Yes, now also a bit shorter.